src/test/ui/79040.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
```

Comment:
This file name should be `issue-79040.rs` and can be placed in `ui/typeck` or `ui/consts`, I think.